includes/SiteGen/SiteGen.php

```diff
@@ -823,11 +844,13 @@ function ( $key ) use ( $menu_patterns_slugs ) {
 						'hiivetoken' => HiiveConnection::get_auth_token(),
 						'prompt'     => array(
 							'site_description' => $site_description,
+							'keywords'   => wp_json_encode( $keywords ),
 							'content_style'    => wp_json_encode( $content_style ),
 							'target_audience'  => wp_json_encode( $target_audience ),
 						),
 						'page'       => $page,
-						'keywords'   => wp_json_encode( $keywords ),
+						'primaryType' => $primaryType,
```

Comment:
We might need corresponding changes on the worker to read these from here and then pass it on to the content-gen-proxy

That worker needs changes
- https://github.com/newfold-labs/cf-worker-ai-sitegen/blob/main/src/routes.js#L48
- https://github.com/newfold-labs/cf-worker-ai-sitegen/blob/main/src/services/sitegen.js#L10

Reply:
Yes, I'm working on that.